### DISCUSSION TITLE
Option to Open "View Page" in the Same Tab After Publishing

### DISCUSSION THREAD

**Discussion by JennyPower** [elementor/elementor#28856] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

Currently, when using Elementor, the "View Page" option after publishing or updating a page opens the page in a new tab by default. While this allows users to continue editing without losing their place, it can disrupt the workflow for those who prefer to view changes in the same tab.
For users who frequently switch between editing and previewing, having the option to open the "View Page" in the same tab would streamline their workflow and reduce the number of open tabs, leading to a more efficient editing process.

### Describe the solution you'd like.

I propose adding an option in the Elementor settings to allow users to choose whether the "View Page" action opens in the same tab or a new tab. This would provide greater flexibility and accommodate different user preferences.

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by FlourishDesignStudio** [comment]
Yes, I agree! It can also result in having a LOT of tabs open, and often with the same page. This could end up being confusing and editing the wrong version if not careful. PLEASE make this tweak!!!

**Comment by jibap** [comment]
Yes, yes, yes !! 
[Here are my suggestions](https://www.reddit.com/r/elementor/comments/1gek5ds/comment/lv2h7tn/?context=3) to enhance Elementor in future versions !
